### PR TITLE
RPM common packages optional scons and build fixes

### PR DESCRIPTION
**Pull request by tst-ppenev**
* Made compiling Open GEE as a dependency of the RPM build tasks optional. (Can be enabled from the command line.)

* Rolled back to a version of OS Package that produces a `common` RPM which installs on Cent OS 6.

* Removed the `.gradle` directory.

### CODE REVIEW

**Comment by tst-dsheremata** [comment]
I've tested and it looks good.  Small fixes will be needed as part of the subsequent server and fusion RPM stories, as agreed to at scrum.  (These should also include the doc updates regarding build environment and documentation updates. )

File: earth_enterprise/rpms/build.gradle [hunk 1]
```diff
@@ -1,17 +1,26 @@
-/*
-    See documentation on the osPackage plugin here: https://github.com/nebula-plugins/gradle-ospackage-plugin
+// Command-line parameters:
+//
+//     ./gradlew -PbuildOpenGee:
+// Compile and install Open GEE in a temporary directory before packaging.
+// By default no compilation, or installation are run.  In that case, you must
+// ensuce the main scons "stage_install" target has been called before building
```
Comment (tst-lsavoie) [inline earth_enterprise/rpms/build.gradle]: I'm pretty sure 'ensuce' is not a word.

File: earth_enterprise/rpms/build.gradle [hunk 2]
```diff
@@ -1,17 +1,26 @@
-/*
-    See documentation on the osPackage plugin here: https://github.com/nebula-plugins/gradle-ospackage-plugin
+// Command-line parameters:
+//
+//     ./gradlew -PbuildOpenGee:
+// Compile and install Open GEE in a temporary directory before packaging.
+// By default no compilation, or installation are run.  In that case, you must
```
Comment (tst-lsavoie) [inline earth_enterprise/rpms/build.gradle]: This line is confusing to me.  The previous line says that this command compiles and installs Open GEE, and this line says that it doesn't.  I think maybe most of this paragraph is meant to apply to the case when you don't specify `-PbuildOpenGEE`, but that wasn't really clear to me.

**Comment by tst-lsavoie** [comment]
If you just run `./gradlew` nothing happens.  We either need to document the task to run or add some default tasks.

File: earth_enterprise/rpms/opengee-common/post-install.sh [hunk 3]
```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright 2017 Google Inc.
```
Comment (tst-ccamp) [inline earth_enterprise/rpms/opengee-common/post-install.sh]: This is a new file, so shouldn't it be copyright "Open GEE Contributors"?

**Comment by tst-lsavoie** [comment]
On the commits by other authors: @tst-dsheremata has approved this PR and @tst-eclamar has submitted his commits in a separate PR, so both are OK with their commits being included in this repo.

**Comment by tst-lsavoie** [comment]
Also, I'm OK with my commits being included.

**Pull request merged**